pkg/reduce/data_forward_test.go

```diff
@@ -184,12 +184,12 @@ func (m MaxReduceTest) ApplyReduce(_ context.Context, partitionID *partition.ID,
 	for msg := range messageStream {
 		var payload PayloadForTest
 		_ = json.Unmarshal(msg.Payload, &payload)
-		if max, ok := maxMap[msg.Key]; ok {
+		if max, ok := maxMap[msg.Keys[0]]; ok {
```

Comment:
we should modify map to have []string as the key

Reply:
the map cannot have []string as a key, so used the first element of the array in the tests.

Reply:
ah yes, these are slices.